src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -1 +1,117 @@
+import com.google.cloud.translate.TranslateOptions;
+import com.google.cloud.translate.Translation;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** An item on a todo list. */
+
+/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+@WebServlet("/data")
+public class DataServlet extends HttpServlet {
+  private final List<Object> json = new ArrayList<>();
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("Comments").addSort("score-class", SortDirection.ASCENDING);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    for (Entity entity : results.asIterable()) {
+      String comment = (String) entity.getProperty("comments-class");
+      json.add(comment);
+    }
+
+    // Send JSON string.
+    String jsonVersion = new Gson().toJson(json);
+    response.setContentType("application/json;");
+    response.getWriter().println(jsonVersion);
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get written feedback.
+    String classFeedback = request.getParameter("class-input");
+    String professorFeedback = request.getParameter("prof-input");
+    // Get class/professor ratings.
+    String ratingClass = request.getParameter("rating-class");
+    String ratingProfessor = request.getParameter("rating-professor");
+    boolean translateEnglish = Boolean.parseBoolean(request.getParameter("languages"));
+
+    if (translateEnglish) {
+      Translate translate = TranslateOptions.getDefaultInstance().getService();
+      Translation translationClassFeedback =
+          translate.translate(classFeedback, Translate.TranslateOption.targetLanguage("en"));
+      Translation translationProfessorFeedback =
+          translate.translate(professorFeedback, Translate.TranslateOption.targetLanguage("en"));
+      String translatedClassFeedback = translationClassFeedback.getTranslatedText();
+      String translatedProfessorFeedback = translationProfessorFeedback.getTranslatedText();
+      classFeedback = translatedClassFeedback;
+      professorFeedback = translatedProfessorFeedback;
+    }
+
+    Document docClass =
+        Document.newBuilder().setContent(classFeedback).setType(Document.Type.PLAIN_TEXT).build();
+    Document docProfessor =
+        Document.newBuilder()
+            .setContent(professorFeedback)
+            .setType(Document.Type.PLAIN_TEXT)
+            .build();
+
+    LanguageServiceClient languageServiceForClass = LanguageServiceClient.create();
+    Sentiment sentimentForClass =
```

Comment:
This is fine but considering you have "scoreClass" below I think it would read better and more more consistent if this was changed to "classSentiment". Similar below